frontend/public/locales/en/translation.json

```diff
@@ -2718,19 +2718,19 @@
     "template.create": "Create Ansible template",
     "template.create.config.wizard.title": "Configure Ansible job template name and Ansible credential",
     "template.create.destroy": "Destroy",
-    "template.create.destroy.wizard.title": "Configure Ansible job templates to run for your cluster at destroy",
+    "template.create.destroy.wizard.title": "Configure automation templates to run for your cluster at destroy",
```

Comment:
"Automation template" = ClusterCurator, so it is a template that contains lists of Ansible templates (both Ansible job templates and Ansible workflow job templates) from which Ansible jobs will be created before or after various cluster lifecycle events. So I don't think this is the correct term here.

Reply:
I thought you wanted to address terminology separately, so if that is the case, please revert all changes to translation.json other than the new strings. I know this unfortunately affects tests as well.